layouts/partials/head.html

```diff
@@ -99,6 +99,10 @@
           success: function(res) {
             $.each(res.data, function(index, item) {
               var description=item['product'] + ' ' + item['version'] + ' cluster on ' + item['cloud'];
+              if (item['error'] !== "") {
+                  $("#ul_clusters").append('<li>' + item["cluster"] + ': ' + item['error'] + '</li>');
```

Comment:
Right now, the value of `item['error']` is always "cannot reach cluster", but even so, maybe it's better to show some set message here (instead of `item['error']`)?

Reply:
I like it like this. We can tune the message on the backend if it needs to change. Most likely "connot reach cluster" will always be fine.